scripts/query/query.ts

```diff
@@ -0,0 +1,41 @@
+import type { SnapManifest } from '@metamask/snaps-utils';
+import fs from 'fs';
```

Comment:
If you import from `fs/promises`, you can just use `fs.readFile` instead of `fs.promises.readFile`.
```suggestion
import fs from 'fs/promises';
```

Reply:
Fixed.